src/Native/Runtime/inc/rhbinder.h

```diff
@@ -846,95 +841,6 @@ enum RhEHClauseKind
 
```

Comment:
You can delete ModuleHeader from this file, and WellKnownMethods.h / WellKnownMethodList.h. They go together with Module.* that this change is deleting.

Reply:
Done!